### PR TITLE
Cooking QOL + code cleanup

### PR DESCRIPTION
**Pull request by cranberrypursuit**
## About The Pull Request
Yeah so cooking currently doesn't give you any xp, plus it's frustratingly slow to do.
This PR aims to address both.

Goals:
- [x] Provide XP when cooking
- [x] Speed up all the do_afters and ensure they reflect the cooking skill
- [x] Clean up the code oh my god there's so many span class= aaaaaaaaaaaaaaa

## Why It's Good For The Game
I'll fill this out once I've got it down

Cooking is slow and frustrating as it stands. No XP is given for actually cooking so you're stuck at a snails pace.
This rewards players for cooking with decreased time for each step.
Also standardises some of the nutrition values from food (hard tack was on the same level as a loaf of bread??)

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
span_notice("Might it also be worth it to give folk cooking xp for chopping/kneading/etc? Just a small detail :3 either way cooking code needed a shlappin so this is awesome"))

File: modular/Neu_Food/code/raw/NeuFood_processed.dm [hunk 1]
```diff
@@ -13,21 +13,23 @@
 	icon_state = "fat"
 	list_reagents = list(/datum/reagent/consumable/nutriment = SNACK_DECENT)
 	eat_effect = /datum/status_effect/debuff/uncookedfood
-/obj/item/reagent_containers/food/snacks/fat/attackby(obj/item/I, mob/user, params)
+
+/obj/item/reagent_containers/food/snacks/fat/attackby(obj/item/I, mob/living/user, params)
 	var/found_table = locate(/obj/structure/table) in (loc)
 	if(user.mind)
 		short_cooktime = (60 - ((user.mind.get_skill_level(/datum/skill/craft/cooking))*5))
```
Comment (noelle-lavenza) [inline modular/Neu_Food/code/raw/NeuFood_processed.dm:20]: I feel like all of these should be more like...
```suggestion
		short_cooktime = (6 SECONDS) - (user.mind.get_skill_level(/datum/skill/craft/cooking) * (0.5 SECONDS))
```
which makes it clearer how it works—it's 6 seconds to begin with, and each cooking level reduces it by half a second.
Also I would probably take care to make sure it doesn't drop below 0? I don't want to know what happens if you give a do_after a negative delay.

**Comment by CosmicLAER** [comment]
WOOOOO LET'S GOOOOOOOOO

File: modular/Neu_Food/code/raw/NeuFood_processed.dm [hunk 2]
```diff
@@ -13,21 +13,23 @@
 	icon_state = "fat"
 	list_reagents = list(/datum/reagent/consumable/nutriment = SNACK_DECENT)
 	eat_effect = /datum/status_effect/debuff/uncookedfood
-/obj/item/reagent_containers/food/snacks/fat/attackby(obj/item/I, mob/user, params)
+
+/obj/item/reagent_containers/food/snacks/fat/attackby(obj/item/I, mob/living/user, params)
 	var/found_table = locate(/obj/structure/table) in (loc)
 	if(user.mind)
-		short_cooktime = (60 - ((user.mind.get_skill_level(/datum/skill/craft/cooking))*5))
+		short_cooktime = (6 SECONDS) - (user.mind.get_skill_level(/datum/skill/craft/cooking) * (0.5 SECONDS))
```
Comment (noelle-lavenza) [inline modular/Neu_Food/code/raw/NeuFood_processed.dm:20]: for what it's worth I meant doing it for all of them, not just the one I marked

**Pull request merged**